package.json

```diff
@@ -7,8 +7,8 @@
     "build": "webpack --mode production",

```

Comment:
don't forget to change main to index.tsx.

Reply:
Changed extension to .tsx.